src/Protego.sol

```diff
@@ -1,116 +1,129 @@
         return keccak256(abi.encode(_usr, _tag, _fax, _eta));
     }
 
-    // Calculate the plan id of a conformant DssSpell
-    function id(DSSpellLike _spell) public view returns (bytes32) {
+    /// @notice Calculates the id of a conformant `DssSpell`.
+    /// @param _spell The spell address.
+    function id(DsSpellLike _spell) public view returns (bytes32) {
         return id(_spell.action(), _spell.tag(), _spell.sig(), _spell.eta());
     }
 
-    // Return true if a plan matching the set of attributes is currently planned
-    function planned(address _usr, bytes32 _tag, bytes memory _fax, uint256 _eta) public view returns (bool) {
+    /// @notice Returns whether a plan matching the set of attributes is currently planned.
+    /// @param _usr The address lifted to the hat.
+    /// @param _tag The tag identifying the address.
+    /// @param _fax The encoded call to be made in `_usr`.
+    /// @param _eta The expiry date.
+    function planned(address _usr, bytes32 _tag, bytes memory _fax, uint256 _eta) external view returns (bool) {
         return planned(id(_usr, _tag, _fax, _eta));
     }
 
-    // Return true if a conformant DssSpell is planned
-    function planned(DSSpellLike _spell) public view returns (bool) {
+    /// @notice Returns whether aspell is planned or not.
+    /// @param _spell The spell address.
+    function planned(DsSpellLike _spell) external view returns (bool) {
         return planned(id(_spell));
     }
 
-    // Return true if an id is planned
+    /// @notice Returns whether an id is planned or not.
+    /// @param _id The ide of the plan.
     function planned(bytes32 _id) public view returns (bool) {
-        return DSPauseLike(pause).plans(_id);
+        return DsPauseLike(pause).plans(_id);
     }
 
-
-    // Permissionlessly block everything
-    // Note: In some cases, due to a governance attack or other unforseen
-    //       causes, it may be necessary to block any spell that is entered
-    //       into the pause proxy. In this extreme case, the system can be
-    //       protected during the pause delay by lifting the Protego contract
-    //       to the hat role, which will allow any user to permissionlessly
-    //       drop any id from the pause.
-    //       This function is expected to revert if it does not have the
-    //       authority to perform this function.
+    /// @notice Permissionlessly drop anything.
+    /// @dev In some cases, due to a governance attack or other unforseen causes, it may be necessary to block any spell
+    ///      that is entered into the pause proxy.
+    ///      In this extreme case, the system can be protected during the pause delay by lifting the Protego contract to
+    ///      the hat role, which will allow any user to permissionlessly drop any id from the pause.
+    ///      This function is expected to revert if it does not have the authority to perform this function.
+    /// @param _usr The address lifted to the hat.
```

Comment:
I think using `The address lifted to the hat` to explain `_usr` is misleading as `drop` could only be called when the protego itself is lifted to the hat. So by that explanation, it's the protego address. It might be better to use less time-dependent definition, e.g. `Address of the scheduled spell` or `Address of the plotted spell`

Reply:
Updated.